### ISSUE TITLE
Unable to pass Authorization header

### ISSUE THREAD

**Issue by JulianJJ** [dapr/dotnet-sdk#1352]
## Expected Behavior

When I pass Authorization header I should receive in the target API just like dapr api key

## Actual Behavior

Authorization header is not received in the target API

**Comment by WhitWaldo** [comment]
Could you share some more information about what you're calling and how? Bonus points if you could share a simple repro project!

**Comment by WhitWaldo** [comment]
@JulianJJ I put together my own sample project and was unable to reproduce what you're seeing. Again, your own repro would be helpful, but in lieu of that, could you answer a couple of questions for me:
- Could you confirm that you're using `InvokeMethodAsync` and passing it an `HttpRequestMessage` decorated with the Authorization header?
- Does the receiving API receive the request at all?
- Do you see any headers on the target API? Is it only missing the Authorization header?

**Comment by WhitWaldo** [comment]
@JulianJJ Thank you for the additional details. I'll circle back should I need any other information.

**Comment by WhitWaldo** [comment]
@JulianJJ Were you able to put a repro project together for this bug?

**Comment by JulianJJ** [comment]
@WhitWaldo Not yet.. Will let you know soon

**Comment by WhitWaldo** [comment]
@JulianJJ Circling back here - is this still something you're experiencing an issue with?

**Comment by susin03** [comment]
As additional info, I'm running the apps with dotnet Aspire with this code:

builder.AddProject<Projects.ProjectA>("projectA")
    .WithDaprSidecar(appId: "projectA");

builder.AddProject<Projects.ProjectB>("projectB")
    .WithDaprSidecar(appId: "projectB");

**Comment by susin03** [comment]
Hey! So, the following happened: I debugged the whole request path, and was able to observe that the headers were still present during the last interaction in my AspNet app.
When the request is made via Dapr (Windows, running in Docker) the Authorization Header is removed.

To be sure of that, I just deployed my app in my Kuberntes environment and the same AspNet code worked as intended.

I don't know how to proceed with the issue now. Thanks in advance.

**Comment by JulianJJ** [comment]
Yes that is correct and I think its not yet fixed. This is very problematic.

**Comment by WhitWaldo** [comment]
> Hey! So, the following happened: I debugged the whole request path, and was able to observe that the headers were still present during the last interaction in my AspNet app. When the request is made via Dapr (Windows, running in Docker) the Authorization Header is removed.
> 
> To be sure of that, I just deployed my app in my Kuberntes environment and the same AspNet code worked as intended.
> 
> I don't know how to proceed with the issue now. Thanks in advance.

When do you mean the headers were present - in other words, did you see that the headers were intact when the request left your application via the SDK and were not present when forwarded from the runtime? What I'm getting at is trying to suss out if this is an SDK or runtime issue.
